### DISCUSSION TITLE
Se me cuelga git al colocar el comando "npm test JSII.test.js"

### DISCUSSION THREAD

**Discussion by DevCCristian** [atralice/Curso.Prep.Henry#170] [General]
Ya resolví el segundo ejercicio de JavaScript pero al mandar el comando "npm test JSII.test.js" para revisar el test se queda colgado, me he quedado esperando como 30 mnts pero no pasa para verificar las respuestas.
Porfavor ayuda. gracias

**Comment by criszam99** [comment]
Probá con npm test 02.
No te olvides antes de hacer Ctrl S para guardar los cambios!
Lic. Cristina del V. Zamorano

**Comment by Rodrybnz** [comment]
Estoy con el mismo problema. ¿Como lo solucionaste?

**Comment by adonayre1** [comment]
Buenos días, me pueden ayudar necesito contactarme con Cristina Zamorano
